manageiq-client is incompatible with python 2.7

All manageiq ansible modules in combination with python 2.7 fail with the error 'AttributeError: 'module' object has no attribute 'monotonic'. This is cause by the required manageiq-client module that no longer can be created in pyhton 2.x. https://pypi.org/project/manageiq-client/ states manageiq-client is compatible with 2.7

```
==================================== ERRORS ====================================
_ ERROR collecting test/units/modules/remote_management/manageiq/test_create_tenant.py _
test/units/modules/remote_management/manageiq/test_create_tenant.py:7: in <module>
    from ansible.modules.remote_management.manageiq import manageiq_tenant
lib/ansible/modules/remote_management/manageiq/manageiq_tenant.py:172: in <module>
    from ansible.module_utils.manageiq import ManageIQ, manageiq_argument_spec
lib/ansible/module_utils/manageiq.py:37: in <module>
    from manageiq_client.api import ManageIQClient
venv2/local/lib/python2.7/site-packages/manageiq_client/api.py:14: in <module>
    from wait_for import wait_for
venv2/local/lib/python2.7/site-packages/wait_for/__init__.py:22: in <module>
    get_time = time.monotonic
E   AttributeError: 'module' object has no attribute 'monotonic'
```

The used module wait_for (https://pypi.org/project/wait-for/) is python 3 only.

--- a/setup.py
+++ b/setup.py
@@ -2,8 +2,17 @@
 # -*- encoding: utf-8 -*-
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+import sys
 from setuptools import setup
 
+
+if sys.version_info < (3,):
+    versioned_requirements = ["wait-for<1.1.0"]
+else:
+    versioned_requirements = ["wait-for"]
+
+requirements = ["requests", "iso8601", "simplejson", "six"]
+requirements.extend(versioned_requirements)
 
 setup(
     name="manageiq-client",
@@ -16,13 +25,7 @@ setup(
     url="https://github.com/ManageIQ/manageiq-api-client-python",
     packages=["manageiq_client"],
     package_dir={'': 'src'},
-    install_requires=[
-        'requests',
-        'wait-for',
-        'iso8601',
-        'simplejson',
-        'six',
-    ],
+    install_requires=requirements,
     setup_requires=[
         'setuptools_scm',
     ],

